### ISSUE TITLE
"Only alert once" but per time period

### ISSUE THREAD

**Issue by wapsi** [SchildiChat/SchildiChat-android#234]
**Is your feature request related to a problem? Please describe.**
"_Only alert once_" is a killer feature that does not exist in the upstream Element project. It's very useful when you have chats where the activity is very high by time to time. No more constant phone beeping, even if you want to keep noisy notifications turned on for specific chat. But even when "_Only alert once_" setting is enabled, sometimes you don't use your phone for several hours and the noisy notifications for the chats remain silent, even the discussion there has been silent for an hour and now a new message arrives.

**Describe the solution you'd like**
It would be really nice if "_Only alert once_" feature would have an option to fine tune the timeout for the muting, like "_Only alert once_", but it's active for tunable amount of minutes?

**Describe alternatives you've considered**
Some Android versions (like LineageOS) have built-in "silent repetitive notifications for 10/30/60 minutes", but my Samsung's Android does not have this feature.

**Comment by SpiritCroc** [comment]
Agreed that would be a cool feature. Currently the only-alert-once setting just uses something the notification library gives us for free, need to investigate how much effort it would be to do based on the last notification timestamp.
